alf/utils/value_ops_test.py

```diff
@@ -134,4 +135,7 @@ def test_generalized_advantage_estimation(self):
 
 
 if __name__ == '__main__':
-    tf.test.main()
+    from alf.utils.common import set_per_process_memory_growth
+
+    set_per_process_memory_growth()
+    unittest.main()
```

Comment:
DiscountedReturnTest is subclass of tf.test.TestCase. Does it work with python unittest?

Reply:
yes, it works